Eukalyptus-Dreizack/dcc
# CLI-Base and automatic npm deployment

## Description
This adds a small repository base and automatic npm deployment

## Related issue
Solves #32 

## How can this be tested?
`pnpm run bns dccmakerbundle:info:description`
Should print out the description of the JSON. Also if we merge in master it should auto deploy to npm.
